specificatie/kadaster-natuurlijk-personen.yaml

```diff
@@ -53,7 +53,7 @@ paths:
     get:
       operationId: GetKadasterPersonen
       description: |
-                    Het zoeken van bij het kadaster geregistreerde natuurlijke personen die niet in de basisregistratie personen (BRP) zijn ingeschreven (of wel zijn ingeschreven maar niet zijn _gematched_ bij het inschrijven van de akte). Kadasternatuurlijkpersonen worden niet geactualiseerd.
+                    Zoek bij het kadaster geregistreerde natuurlijke personen die niet in de basisregistratie personen (BRP) zijn ingeschreven, of wel zijn ingeschreven maar niet gekoppeld bij het inschrijven van de akte. Kadasternatuurlijkpersonen worden niet geactualiseerd.
```

Comment:
Elke zin in een eigen regel? Dit gebeurt soms wel en soms niet. Is wel makkelijker bij het lezen. Dan hoeft je niet heen en weer te scrollen